xmeffects.js

```diff
@@ -287,3 +293,4 @@ var effects_t1 = [  // effect functions on tick 1+
   eff_unimplemented,  // y
   eff_unimplemented   // z
 ];
+})(window || {});
```

Comment:
is `|| {}` this for unit tests or something, so it can run without an actual `window`?